charts/tempo-distributed/values.yaml

```diff
@@ -705,17 +705,38 @@ config: |
         kvstore:
           store: "memberlist"
 
-  {{- if .Values.minio.enabled }}
+  auth:
+    type: enterprise
+
+  http_api_prefix: {{get .Values.tempo.structuredConfig "http_api_prefix"}}
+
   admin_client:
     storage:
-      backend: s3
+      type: {{.Values.storage.admin.backend}}
+      {{- if .Values.minio.enabled }}
```

Comment:
What do you think about not special casing the minio here?  In my local config snippet from a couple months ago, I had just specified the values directly in my values files, which I think is why it worked for me.